features/features.go

```diff
@@ -0,0 +1,96 @@
+//go:generate stringer -type=FeatureFlag
+
+package features
+
+import (
+	"expvar"
+	"fmt"
+	"sync"
+)
+
+type FeatureFlag int
+
+const (
+	unused                    FeatureFlag = iota
+	DNSAllowLoopbackAddresses             // Common
+	CheckMalformedCSR                     // WFE
+	DoNotForceCN                          // RA, CA
+	ReuseValidAuthz                       // RA
+	EnableMustStaple                      // CA
+	AcceptRevocationReason                // WFE
+)
+
+// List of features and their default value
```

Comment:
But we solved that by calling the flag `DoNotForceCN`. I think it's probably better to solve this problem with slightly awkward flag names than with extra code.